src/locker.rs

```diff
@@ -0,0 +1,112 @@
+use std::ops::{Deref, DerefMut};
+
+use crate::isolate::Isolate;
+use crate::scope::data::ScopeData;
+use crate::support::Opaque;
+
+#[repr(C)]
+#[derive(Debug)]
+struct LockerHandle(Opaque);
```

Comment:
unused

Reply:
Update: Remove the unused `LockerHandle`.